baselines/FedPM/FedPM/conf/base.yaml

```diff
@@ -0,0 +1,54 @@
+---
+# this is the config that will be loaded as default by main.py
+# Please follow the provided structure (this will ensuring all baseline follow
+# a similar configuration structure and hence be easy to customise)
+
+dataset:
+  name: 'mnist' # [cifar10, mnist]
+  split: 'iid' # {iid, non-iid}
+  classes_pc: 4
+  minibatch_size: 64 #32
+
+model:
+  name: 'LeNet' # [Conv8/6/4, LeNet, Mask, Conv6]
+  mode: 'dense' # [mask, dense]
+  local_epochs: 3
+  activation: 'relu'
+  agg_steps: 5
+
+strategy:
+  _target_: # points to your strategy (either custom or exiting in Flower)
+  fedpm:
+    local_epochs: 3
+    local_lr: 0.1
+
+  fedavg:
+    local_lr: 0.0003
+    server_lr: 0.1
+
+  dense:
+    local_epochs: 3
+
+  sign_sgd:
+    local_lr: 0.01
+    server_lr: 0.0002
+
+client:
+  local_epochs: 3
+  local_lr: 0.1
+
+simulation:
+  fraction_fit: 0.2
+  fraction_evaluate: 0
+  n_clients: 10
+  n_rounds: 200
+  verbose: 0
+  store_output: True
+  strategy: 'fedpm' # [fedpm, dense]
+
+ray_init_args:
+  ignore_reinit_error: True
+  include_dashboard: True
+  num_cpus: 6
+  num_gpus: 1
```

Comment:
what these do is expose only 6CPUs and 1GPU to the whole VirtualClientEngine. I'm not sure if this is what you wanted to do. 

If the idea was to set those as the `client_resources`, please define them separately in the config:
```
client_resources:
    num_cpus: 6
    num_gpus: 1
```
Note that depending on your model and training costs, 1 whole GPU for a single client might result in the GPU being underutilised. If that's the case you can pack more clients in the same GPU by lowering `num_gpus`. More info in the docs: https://flower.dev/docs/framework/how-to-run-simulations.html#virtualclientengine-resources